routes/web.public.php

```diff
@@ -2,12 +2,15 @@
 
 declare(strict_types=1);
 
+use App\Http\Controllers\iDealController;
 use Illuminate\Support\Facades\Route;
 use Updater\Http\Controllers\UpdateController;
 
 // iDeal integration
-Route::post('iDeal-webhook', 'iDealController@webhook');
-Route::get('iDeal-response/{participant}/{event}', 'iDealController@response');
+Route::post('iDeal-webhook', [iDealController::class, 'webhook']);
+Route::get('iDeal-response/{participant}/{event}', [iDealController::class, 'eventPaymentResponse']);
+Route::get('iDeal-response', [iDealController::class, 'genericResponse']);
+Route::get('doneer', 'DonateController');
```

Comment:
Nav van wat ik zei in de app, mss kun je deze vast in `routes/api.php` zetten?

Reply:
Nee, doe nog maar niet eigenlijk. Ik zit in mijn eigen PR hier wat mee te kloten en er moeten op andere plekken ook dingen aangepast worden voor goede routing. Dus laat hier maar zitten, ik downmerge en fix het wel.